mlflow/server/js/src/components/artifact-view-components/ShowArtifactImageView.js

```diff
@@ -24,6 +24,7 @@ class ShowArtifactImageView extends Component {
     this.fetchImage();
   };
 
+  // The CI should fail because this code violates arrow-parens='always'.
   componentDidUpdate = prevProps => {
     if (prevProps.path !== this.props.path) {
       this.fetchImage();
```

Comment:
The lint error detected here is **fixable**, so `eslint src --fix` exits with 0.